disperser/apiserver/server.go

```diff
@@ -285,7 +285,7 @@ func (s *DispersalServer) disperseBlob(ctx context.Context, blob *core.Blob, aut
 
 	// If paymentHeader is not empty, we use the meterer, otherwise we use the ratelimiter if the ratelimiter is available
 	if paymentHeader != nil && *paymentHeader != (core.PaymentMetadata{}) {
-		err := s.meterer.MeterRequest(ctx, *blob, *paymentHeader)
+		err := s.meterer.MeterRequest(ctx, *paymentHeader, uint(blobSize), blob.GetQuorumNumbers())
```

Comment:
I think this needs to be converted from byte length to symbol length (https://github.com/Layr-Labs/eigenda/blob/db3da32518159617d301128d660362dc3b05b2a5/encoding/utils.go#L10)

Reply:
thank for the link, I've updated to use the conversion